### PR TITLE
User profile image

### PR DESCRIPTION
**Pull request by ghost**
Changed facebook and gravatar template tags with account tag in user.html - correct user profile picture is displayed again.

### CODE REVIEW

File: piplmesh/frontend/templates/user/user.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "plain.html" %}
 
-{% load i18n facebook gravatar %}
+{% load i18n account staticfiles %}
```
Comment (mitar) [inline piplmesh/frontend/templates/user/user.html]: Why `staticfiles`?

Comment (ghost) [inline piplmesh/frontend/templates/user/user.html]: whoops, c/p error

Comment (mitar) [inline piplmesh/frontend/templates/user/user.html]: Whaaat? You are admitting to copy-pasting in front of my eyes? Be careful!!!

Comment (ghost) [inline piplmesh/frontend/templates/user/user.html]: I hope you forgive me because of a quick fix :)

**Comment by mitar** [comment]
OK.

**Pull request merged**